examples/using_sunkit_magex_pfss/overplotting_onto_aia.py

```diff
@@ -112,22 +97,20 @@
 ###############################################################################
 # Trace field lines from the footpoints defined above.
 
-tracer = tracing.FortranTracer()
+tracer = tracing.PythonTracer()
```

Comment:
I had to do this as the example is broken with it. 

TODO: FIGURE THIS OUT

Reply:
Oh no its due to not my changes: https://docs.sunpy.org/projects/sunkit-magex/en/latest/generated/gallery/using_sunkit_magex_pfss/plot_aia_overplotting.html#sphx-glr-generated-gallery-using-sunkit-magex-pfss-plot-aia-overplotting-py 

I assume astropy+mpl changes have in combo changed this?
https://docs.sunpy.org/projects/sunkit-magex/en/stable/generated/gallery/using_sunkit_magex_pfss/plot_aia_overplotting.html stable was ok when we did the release, so the only change is now updates upstream

Reply:
sunpy 5.1.4 to sunpy 6.0 is what broke this. 

There is a bug somewhere in sunpy we need to fix.